### ISSUE TITLE
v0.18.3 --target typescript-modern generates `.js` file extensions

### ISSUE THREAD

**Issue by rosskevin** [apollographql/apollo-codegen#335]
Command:

```
apollo-codegen generate src/**/*.graphql --schema schema.json --target typescript-modern
```

It does still have `/* tslint:disable */` in the file, so I assume it isn't using the flow type generator.

**Comment by rosskevin** [comment]
Fixed in #334

**Comment by patrick91** [comment]
has this been releases? :)
